src/Adapter/Category/CommandHandler/DeleteCategoryMenuThumbnailImageHandler.php

```diff
@@ -92,7 +92,7 @@ public function handle(DeleteCategoryMenuThumbnailImageCommand $command)
 
         $thumbnailPath = sprintf(
             '%s%s-%s_thumb.jpg',
-            $this->configuration->get('_PS_CAT_IMG_DIR_'),
+            _PS_CAT_IMG_DIR_,
```

Comment:
Woah that is a nasty bug that you have found 😮(and fixed)

So it means the feature "delete category menu thumbnail image" was broken ?

Reply:
Actually I refactored code several times searching for a good solution how to imitate file upload; And this code I've change was created by me.

Reply:
It wasn't a bug: https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/Configuration.php#L101

Reply:
😆 @PierreRambaud thank you legacy configuration with fallback behavior 🙏

Reply:
That's bad refactoring style from me. Thank you for showing it to me :slightly_smiling_face: